src/libopensc/card-epass2003.c

```diff
@@ -1157,6 +1163,15 @@ epass2003_init(struct sc_card *card)
 	else
 		exdata->smtype = KEY_TYPE_DES;
 
+	if (0x84 == data[14])
+	{
```

Comment:
Again, please put the curly brackets in the same line as the if statement to be consistent with the rest of this file.